_shared/project/Makefile

```diff
@@ -131,9 +131,9 @@ $(call help,make sure,"make sure that the formatting$(comma) linting and tests a
 sure: python
 sure:
 {% if cookiecutter._directory in ['pyapp', 'pyramid-app'] %}
-	@pyenv exec tox --parallel -qe 'checkformatting,lint,tests,coverage,functests'
+	@pyenv exec tox --parallel -qe 'checkformatting,lint,typecheck,tests,coverage,functests'
 {% else %}
-	@pyenv exec tox --parallel -qe 'checkformatting,lint,tests,py{{ "{" }}{{ python_versions()|rest|pyformat(PyFormats.MAJOR_MINOR_FMT)|separator(",") }}{{ "}" }}-tests,coverage,functests,py{{ "{" }}{{ python_versions()|rest|pyformat(PyFormats.MAJOR_MINOR_FMT)|separator(",") }}{{ "}" }}-functests'
+	@pyenv exec tox --parallel -qe 'checkformatting,lint,typecheck,tests,py{{ "{" }}{{ python_versions()|rest|pyformat(PyFormats.MAJOR_MINOR_FMT)|separator(",") }}{{ "}" }}-tests,coverage,functests,py{{ "{" }}{{ python_versions()|rest|pyformat(PyFormats.MAJOR_MINOR_FMT)|separator(",") }}{{ "}" }}-functests'
```

Comment:
Also for packages :+1: As with formatting and linting, we're running a package's typechecking with the package's newest supported version of Python only.

I do wonder if type-checking could create some fun problems when supporting multiple Python versions, or multiple versions of some library, etc.